troncli/h_config.py

```diff
@@ -55,13 +55,21 @@ def __init__(self):
         await self.update_config_store()
 
     async def update_config_store(self):
-        await self.node_list.update_config(self.config_store['nettype'], self.config_store['fullhttpport'], self.config_store['solhttpport'],
-                                     self.config_store['eventhttpport'], self.config_store['fullrpcport'], 
-                                     self.config_store['solrpcport'], self.config_store['eventrpcport'],
-                                     self.config_store['enablememdb'], self.config_store['dbsyncmode'], 
-                                     self.config_store['saveintertx'], self.config_store['savehistorytx'], 
-                                     self.config_store['gridport'], self.config_store['dbname'], 
-                                     self.config_store['dbusername'], self.config_store['dbpassword'])
+        await self.node_list.update_config(self.config_store['nettype'],
```

Comment:
too many parameters to pass
More convenient through (** kwargs)

Reply:
thanks for your suggestion, will optimize it